<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>0</y>
           <width>158</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>(R + G + B) / 3.0</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>610</x>
           <y>0</y>
           <width>205</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0.299R+0.587G+0.114B</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>880</x>
           <y>0</y>
           <width>205</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Subtract histogram</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="MplCanvas" name="raw" native="true"/>
      </item>
      <item>
       <widget class="MplCanvas" name="gray1" native="true"/>
      </item>
      <item>
       <widget class="MplCanvas" name="gray1_hist" native="true"/>
      </item>
      <item>
       <widget class="MplCanvas" name="gray2" native="true"/>
      </item>
      <item>
       <widget class="MplCanvas" name="gray2_hist" native="true"/>
      </item>
      <item>
       <widget class="MplCanvas" name="subtract_hist" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="MplCanvas" name="resize" native="true"/>
      </item>
      <item>
       <widget class="MplCanvas" name="resize_gray" native="true"/>
      </item>
      <item>
       <widget class="MplCanvas" name="resize_hist" native="true"/>
      </item>
      <item>
       <widget class="MplCanvas" name="bright_contrast" native="true"/>
      </item>
      <item>
       <widget class="MplCanvas" name="binary" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <widget class="QSlider" name="resize_slider">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>41</y>
           <width>171</width>
           <height>31</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>10</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Resize</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>700</x>
           <y>10</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Brightness</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QSlider" name="bright_slider">
         <property name="geometry">
          <rect>
           <x>670</x>
           <y>50</y>
           <width>171</width>
           <height>31</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>700</x>
           <y>110</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Contrast</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QSlider" name="contrast_slider">
         <property name="geometry">
          <rect>
           <x>670</x>
           <y>150</y>
           <width>171</width>
           <height>31</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>930</x>
           <y>10</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Binary</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QSlider" name="binary_slider">
         <property name="geometry">
          <rect>
           <x>900</x>
           <y>50</y>
           <width>171</width>
           <height>31</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="equal_btn">
         <property name="geometry">
          <rect>
           <x>910</x>
           <y>120</y>
           <width>161</width>
           <height>51</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Equalization</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplCanvas</class>
   <extends>QWidget</extends>
   <header location="global">mplcanvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
